Arion/ArionCollisionConfig.cmake.in

```diff
@@ -1,10 +1,42 @@
-# Copyright (C) 2017 by Godlike
+# Copyright (C) 2017-2018 by Godlike
 # This code is licensed under the MIT license (MIT)
 # (http://opensource.org/licenses/MIT)
 
 set(PREFIX ${CMAKE_CURRENT_LIST_DIR}/../..)
+set(ARION_LINK_LIBRARIES
+    Epona::Math
```

Comment:
This declaration looks as though there is `Epona::Somethingelse` is there a reason for this syntax?

Reply:
`string::string[::string]` are `ALIAS` targets that can be checked during cmake configuration step. When used, CMake checks if such an ALIAS exists and gives you configuration error if it's not. When present, CMake a target that links an `ALIAS` target inherits its `PUBLIC` and `INTERFACE` target options (link libraries, include directories, etc)

Even when it's the only target in _the namespace_ of Epona, it is still beneficial to use `ALIAS` instead of binary name in order to check target presence and inherit options from the target itself.

https://cmake.org/cmake/help/v3.0/manual/cmake-buildsystem.7.html#alias-targets